bin/main.rb

```diff
@@ -1,3 +1,5 @@
-# frozen_string_literal:true
+#! /usr/bin/ruby
```

Comment:
this line is working :+1: 

it is recommended by microverse to use:

` #!/usr/bin/env ruby`

Reply:
Hi!
I am working by myself because I had problems with my coding partner, and since I am in the middle of Ruby, it would be hard to find a new one.
Willow authorized me to continue by myself.